src/portpicker.py

```diff
@@ -201,8 +201,9 @@ def get_port_from_port_server(portserver_address, pid=None):
             buf = sock.recv(1024)
         finally:
             sock.close()
-    except socket.error:
-        print('Socket error when connecting to portserver.', file=sys.stderr)
+    except socket.error as e:
+        print('Socket error when connecting to portserver:', e,
+              file=sys.stderr)
```

Comment:
My bad.  For some reason I thought file=whatever didn't work even with the future import.